107/src/main/java/org/ehcache/jsr107/EhcacheCachingProvider.java

```diff
@@ -52,6 +52,47 @@
 
   private final Map<ClassLoader, ConcurrentMap<URI, Eh107CacheManager>> cacheManagers = new WeakHashMap<ClassLoader, ConcurrentMap<URI, Eh107CacheManager>>();
 
+  public static class ConfigSupplier {
```

Comment:
this should not be public either. And by usual code style in the project, should be at the end of the file.